CHANGES.md

```diff
@@ -1,3 +1,37 @@
+## edge-20.4.3
+
+This edge release adds functionality to the CLI to output more detail and
+includes changes which support the multi-cluster functionality. Also, the helm
+support has been expanded to make installation more configurable. Finally, the
+HA reliability is improved by ensuring that control plane pods are restarted
+with a rolling strategy
+
+* CLI
+  * Added output to the `linkerd check --proxy` command to list all data
+    plane pods which are not up-to-date rather than just printing the first
+    one it encounters
+  * Added a `--proxy` flag to the `linkerd version` command which lists all
+    proxy versions running in the cluster and the number of pods running
+    each version
+  * Added MeshedPodCount field to TrafficSplit resource rows
+* Controller
+  * Added a rolling update strategy to Linkerd deployments that have
+    multiple replicas during HA deployments to ensure that at most
+    one pod begins terminating before a new pod ready is ready
+* Internal
+  * Added a [security policy](https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository)
+    to facilitate conversations around security
+* Helm
+  * Changed charts to use downwardAPI to mount labels to the proxy container
```

Comment:
This could be `Updated proxy-injector to add pod labels to mount labels for the proxy to attach them as trace attributes` and `Proxy-Injector` can have its own section? or Is it `Controller`?

Reply:
Also, we could added another point i.e `added a new label for the proxy injector to add, linkerd.io/workload-ns which indicates the  namespace of the workload/pod`

Reply:
👍